getting-started/migration-guide.mdx

```diff
@@ -113,6 +113,36 @@ you could [enable caching on Edgee](/services/performance/caching).
 </Note>
 
 
+### Case 4: Edgee is an additional call within your workflow
```

Comment:
I'd try to standardize this section as we name it in 3 different ways:

1. "_Edgee is an additional call within your workflow_" (title)
2. "_Edgee as an additional step in your workflow_" (Frame caption)
3. "_Edgee behind a CDN_" (image alt text)

Personally, I have a slight preference for (1)